src/main/host/shd-process.c

```diff
@@ -473,15 +481,15 @@ static void _process_loadPlugin(Process* proc) {
     /* make sure it has the required init function */
     gpointer symbol = NULL;
 
-    symbol = dlsym(proc->plugin.handle, PLUGIN_MAIN_SYMBOL);
+    symbol = dlsym(proc->plugin.handle, _process_getPluginStartSymbol(proc));
```

Comment:
We could use something like:

```
_process_getPluginStartSymbol(proc) ? _process_getPluginStartSymbol(proc) : PLUGIN_DEFAULT_SYMBOL
```

and then avoid allocating a new string to copy the const string below.